cob_description/urdf/sensors/prosilica.gazebo.xacro

```diff
@@ -25,12 +25,12 @@
         </camera>
         <plugin name="${name}_controller" filename="libgazebo_ros_camera.so">
           <updateRate>10.0</updateRate>
-          <cameraName>${name}_sensor</cameraName>
-          <frameName>/${name}_link</frameName>
+          <cameraName>${name}</cameraName>
+          <frameName>${name}_link</frameName>
           <topicName>${ros_topic}_image_data</topicName>
-          <imageTopicName>/${ros_topic}/image_raw</imageTopicName>
-          <cameraInfoTopicName>/${ros_topic}/camera_info</cameraInfoTopicName>
-          <pollServiceName>/${name}/request_image</pollServiceName>
+          <imageTopicName>${ros_topic}_image_raw</imageTopicName>
+          <cameraInfoTopicName>${ros_topic}_camera_info</cameraInfoTopicName>
```

Comment:
I'd remove `${ros_topic}` completely and hard-code it to `image_raw`/`camera_info` respectively.
We can use `${name}` for setting it to `head_color_camera/left` as `cameraName` is prepended before `imageTopicName` anyway, right?